src/libraries/Common/src/System/Security/Cryptography/Pkcs12Kdf.cs

```diff
@@ -16,14 +16,14 @@ internal static class Pkcs12Kdf
 
         // This is a dictionary representation of the table in
         // https://tools.ietf.org/html/rfc7292#appendix-B.2
-        private static readonly Dictionary<HashAlgorithmName, Tuple<int, int>> s_uvLookup =
-            new Dictionary<HashAlgorithmName, Tuple<int, int>>
+        private static readonly Dictionary<HashAlgorithmName, KeyValuePair<int, int>> s_uvLookup =
```

Comment:
This does not really need to be Dictionary given the number of items and where it is used. Array + linear search would work just fine.

Reply:
I can change it.